### DISCUSSION TITLE
Procedures for collaboration within teams, external parties and the public.

### DISCUSSION THREAD

**Discussion by nicolaasburgers** [nicolaasburgers/definition-library-releases#26] [General]
Hi everyone just thought I'd flag a new page that might be particularly useful in resolving some key angles of curiosity with regard to the DefinitionLibrary plug-in: [Collaboration](https://github.com/nicolaasburgers/definition-library-releases/wiki/Collaboration)

The page is currently only text - images are still to be added to make it easier to read - but it would be great to hear from you as to whether this is clear to you as a reader.  Would a video specifically on this be useful?

**Comment by nicolaasburgers** [comment]
Tagging @amoeamoe, @JBENEITEZ and @sheldon-walters

**Reply by JBENEITEZ** [reply]
Thanks @nicolaasburgers. I've scanned through the article and seems all quite clear to me. But also is worth noting that I've seen and experienced the setup myself, so surely a video or screenshots will help!
